tutorials/physics/using_kinematic_body_2d.rst

```diff
@@ -242,7 +284,53 @@ uses the mouse pointer. Here is the code for the Player, using ``move_and_slide(
 
  .. code-tab:: csharp
 
-    Coming soon
+    using Godot;
+    using System;
+
+    public class KBExample : KinematicBody2D
+    { 
+        PackedScene bullet = (PackedScene)GD.Load("res://Bullet.tscn");
```

Comment:
It's better to make access modifiers explicit, and in this case I guess we better use `private PackedScene _bullet` instead.

And the same goes for `public int Speed`, and `private Vector2 _velocity`.